src/lib/config.ts

```diff
@@ -55,6 +55,9 @@ export function intVar<R>(varName: string, defaultValue?: R): number | R {
 }
 
 export const API_HOST = requiredVar('API_HOST');
+export const API_HEARTBEAT_STATE_TIMEOUT_SECONDS = intVar(
+	process.env.API_HEARTBEAT_STATE_TIMEOUT_SECONDS || '15',
```

Comment:
This should be
```suggestion
	'API_HEARTBEAT_STATE_TIMEOUT_SECONDS', 15,
```
because it takes the var name as the first param, and an optional default value as the second - right now it was trying to assign `process.env[process.env.API_HEARTBEAT_STATE_TIMEOUT_SECONDS || '15']` to `API_HEARTBEAT_STATE_TIMEOUT_SECONDS`